apps/web/src/components/utils/GotAQuestionButton.tsx

```diff
@@ -0,0 +1,13 @@
+import { Button, Size } from '../../design-system';
+
+function onClick() {
+  window?.open('https://discord.novu.co', '_blank')?.focus();
```

Comment:
@p-fernandez I think we like to trigger the intercom chat when they click this button... right @scopsy?

Reply:
Yes, intercom would be best, I think we use react-intercom for that, they have an open method there